CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+0.7.0
```

Comment:
Shouldn't it be 0.6.3, since we do not bump major version in this repo?

Reply:
I think that we use [semver](https://semver.org/) everywhere now